spec/actions_specs/reset_simulator_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    it "raises an exception when no device name is specified" do
+      expect do
+        Fastlane::FastFile.new.parse("lane :test do
+          reset_simulator({})
+        end").runner.execute(:test)
+      end.to raise_error
```

Comment:
This sounds good to me @mgrebenets ! Have you already opened a PR on `fastlane_core`?